modules/sdk-core/src/bitgo/staking/stakingWallet.ts

```diff
@@ -221,16 +221,20 @@ export class StakingWallet implements IStakingWallet {
    * @param transaction
    */
   async prebuildSelfManagedStakingTransaction(transaction: StakingTransaction): Promise<PrebuildTransactionResult> {
-    const builtStakingTransaction = await this.build(transaction);
-    const buildParams = builtStakingTransaction.transaction.buildParams;
+    let buildParams: PrebuildTransactionOptions | undefined;
+    if ((this.wallet.baseCoin.supportsTss() && this.wallet.baseCoin.getFamily() !== 'eth') || this.isEthTss) {
```

Comment:
The description lists `eth like` but we are only checking eth here. Can we generalize this and make it less coin specific?

Reply:
Also a comment on why we need this differentiation in the code would be helpful

Reply:
yeah i meant to write multisig instead of eth like in commit message.

we are checking eth specifically because we have eth multisig and eth tss, so we need 2 separate checks as far as i understand.

Reply:
changed the logic